README.md

```diff
@@ -1,125 +1,170 @@
+
+```shell
+go build act.go
+```
+#### Arcaflow Container Toolkit test and build
+
+```shell
+./act build --build
+```
+
+#### Arcaflow Container Toolkit test, build, and push
+
 ```shell
-./carpenter build --build
+./act build --build --push
 ```
 
+## Arcaflow Container Toolkit as a Package
 
-## Example Build and Push Execution Containerized
+Pull the latest image
 
-### Requirements
+```shell
+docker pull ghcr.io/arcalot/arcaflow-container-toolkit:latest
+```
 
-* docker engine and cli
-* a carpenter image named `carpenter-img`
-* plugin project
-* GitHub username and password
-* Quay username and password
+Run the Arcaflow Container Toolkit image with enviornment variables
 
 ```shell
 docker run \
     --rm \
     -e=IMAGE_TAG="0.0.1"\
     -e=BUILD_TIMEOUT=600\
-    -e=GITHUB_USERNAME=$GITHUB_USERNAME \
-    -e=GITHUB_PASSWORD=$GITHUB_PASSWORD \
     -e=QUAY_USERNAME=$QUAY_USERNAME\
     -e=QUAY_PASSWORD=$QUAY_PASSWORD\
     -e=QUAY_NAMESPACE=$QUAY_NAMESPACE\
     --volume /var/run/docker.sock:/var/run/docker.sock:z \
     --volume $PWD/../arcaflow-plugin-template-python:/github/workspace \
-    carpenter-img build --build --push
+    ghcr.io/arcalot/arcaflow-container-toolkit:latest build --build --push
 ```
-You can override the variables `image_tag` and `image_name` by injecting the environment variable `IMAGE_TAG` `IMAGE_NAME` respectively, set to your chosen string into `carpenter-img` when you run the container.
 
-Additionally, `build_timeout`,`quay_img_exp`, and `quay_custom_namespace` can be overridden.
+## Arcaflow Container Toolkit and Reusable Workflows
 
-`quay_img_exp` is overwritten by injecting the environment variable `QUAY_IMG_EXP`. Configuring this variable from default `never` will update the LABEL added to the image during build time to automatically expire after the time indicated and delete from the repository in Quay. Documentation and time formats can be found [here](https://docs.projectquay.io/use_quay.html#:~:text=Setting%20tag%20expiration%20from%20a%20Dockerfile)
+Arcaflow Container Toolkit can be utilized using the official reusable workflow `arcalot/arcaflow-containter-toolkit/.github/workflows/reusable_workflow.yaml`. 
 
-`quay_custom_namespace` is overwritten by injecting the environment variable `QUAY_CUSTOM_NAMESPACE`. This will overwrite the default value of `""`, and additionally use this requested quay namespace to push the image instead of the inferred namespace automatically assigned by carpenter with `QUAY_NAMESPACE`. This is primarly used when carpenter is being utilized in GitHub Actions CI/CD without needing to change the secrets of the repository where `QUAY_NAMESPACE` value is stored if a different location is required.
+```yaml
+name: Arcaflow Container Toolkit
+on:
+  push:
+    branches:
+      - "**"
+  release:
+    types:
+      - published
+
+jobs:
+  arcaflow-container-toolkit:
+    uses: arcalot/arcaflow-container-toolkit/.github/workflows/reusable_workflow.yaml@main
```

Comment:
Is this supposed to be a repo path? Or should/can it point to a published action?

Reply:
This example is showing the reusable workflow which would yes be a path. The reusable workflow itself uses the published action. Utilizing the reusable workflow in this manner will still allow the user to take advantage of some automatic configurable's as noted in the documentation. If the published action is only required they can call it using the next section `Arcaflow Container Toolkit as an Action'